init.sh

```diff
@@ -57,5 +57,6 @@ else
 fi
 
 # TEMPORARY: fix to lcg releases
-export Gaudi_DIR=/cvmfs/sft.cern.ch/lcg/releases/LCG_96c_LS/Gaudi/v32r2/$platform/
+export Gaudi_DIR=/cvmfs/sft.cern.ch/lcg/releases/Gaudi/v33r1-36bcc/x86_64-centos7-gcc8-opt/
```

Comment:
Hi Valentin, why is this not taken from LCG_97_FCC_2 ?

Reply:
Hi Gerri, this is the version from LCG_97_FCC_2. The other locations all symlink to this directory:
```
/build/vavolkl/FCCSW (release-v0.13 *)$ ls /cvmfs/sft.cern.ch/lcg/releases/LCG_97_FCC_2/Gaudi/v33r1/x86_64-centos7-gcc8-opt -alh
lrwxrwxrwx. 1 cvmfs cvmfs 50 14. Mai 08:50 /cvmfs/sft.cern.ch/lcg/releases/LCG_97_FCC_2/Gaudi/v33r1/x86_64-centos7-gcc8-opt -> ../../../Gaudi/v33r1-36bcc/x86_64-centos7-gcc8-opt
```
but I can also use the path `/cvmfs/sft.cern.ch/lcg/releases/LCG_97_FCC_2/Gaudi/v33r1/x86_64-centos7-gcc8-opt`, then maybe it is clearer from which LCG release this is coming from.

By the way, the issue I'm working around here is that the file `Gaudi.xenv` is not found when just sourcing the view, presumably because it is located in the top level install directory and not symlinked in the view directory. Should I open a ticket?